newsroom/design.py

```diff
@@ -39,11 +50,14 @@ async def detail():
     return await render_template("wire_item.html", item=item, previous_versions=previous_versions)
 
 
-@blueprint.route("/design/")
-async def index():
+@blueprint.endpoint("/design/", auth=False)
+async def index(args: None, params: None, req: Request):
     return await render_template("design_index.html")
 
 
-@blueprint.route("/design/<page>")
-async def page(page):
-    return await render_template("design_%s.html" % page)
+@blueprint.endpoint("/design/<string:page>", auth=False)
+async def page(args: RouteArguments, params: None, req: Request):
+    return await render_template("design_%s.html" % args.page)
```

Comment:
nit:
```suggestion
    return await render_template(f"design_{args.page}.html")
```

Also, we might need to check if the template exists first, otherwise it will be a 500 error. But we can do it in a separate PR.